Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -23,16 +23,19 @@ def __init__(self, *args, **kwargs):
 
         super(Board, self).__init__(*args, **kwargs)
 
-        EXECUTE_IN_PLACE          = 0
+        # 0: QEMU  1: Simics support ->QSP
+        self.ENABLE_SIMICS        = 0
+
+        EXECUTE_IN_PLACE          = 0 if self.ENABLE_SIMICS != 1 else 1
         FREE_TEMP_RAM_TOP         = 0x70000
 
-        self.VERINFO_IMAGE_ID          = 'SB_QEMU '
+        self.VERINFO_IMAGE_ID          = 'SB_QEMU ' if self.ENABLE_SIMICS != 1 else 'SBL_QSP '
         self.VERINFO_PROJ_MAJOR_VER    = 1
-        self.VERINFO_PROJ_MINOR_VER    = 0
+        self.VERINFO_PROJ_MINOR_VER    = 1
         self.VERINFO_SVN               = 1
-        self.VERINFO_BUILD_DATE        = '09/25/2017'
+        self.VERINFO_BUILD_DATE        = '05/03/2021'
 
-        self.BOARD_NAME           = 'qemu'
+        self.BOARD_NAME           = 'qemu' if self.ENABLE_SIMICS != 1 else 'qsp'

```

Comment:
Again, let us use board delta to define different name.

Reply:
Done.